Library/Homebrew/untap.rb

```diff
@@ -0,0 +1,60 @@
+# typed: true
+# frozen_string_literal: true
+
+require "extend/cachable"
+
+module Homebrew
+  # Helpers for the `brew untap` command.
```

Comment:
If these are only used by the `untap` command: can they live in `cmd/untap` instead?

Reply:
I agree that its not the best location but it does make the logic and testing simpler. It'll be easy to move back into the command once we're using the new command interface described in https://github.com/Homebrew/brew/pull/16815.